### PR TITLE
Modify incorrect information in READMD.md

### PR DESCRIPTION
**Pull request by qpakzk**
* Correct the wrong url for Triton Python Client Library

* Correct the wrong path for client.py

### CODE REVIEW

**Comment by Tabrizian** [comment]
Thanks for your contribution. However, I think that the user is able to find out how to use the gRPC client by reading the docs so I don't think it is necessary to include both clients for both of the clients for the examples.

I noticed that you found an incorrect documentation link. Can you please only include that change?

File: README.md [hunk 1]
```diff
@@ -342,8 +342,8 @@ small.
 # Examples
 
 For using the Triton Python client in these examples you need to install
-the [Triton Python Client Library](https://github.com/triton-inference-server/server/blob/master/docs/client_libraries.md#getting-the-client-libraries).
-The Python client for each of the examples is in the `client.py` file.
+the [Triton Python Client Library](https://github.com/triton-inference-server/client#getting-the-client-libraries-and-examples).
```
Comment (Tabrizian) [inline README.md:345]: Please only include this change in your PR.

**Review by Tabrizian — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (qpakzk) [reply README.md:345]: @Tabrizian I updated my commit.

Comment (Tabrizian) [reply README.md:345]: Thanks, can you add "The Python client for each of the examples is in the `client.py` file."  back too?

Comment (qpakzk) [reply README.md:345]: I didn't delete that sentence. You can check it in my commit.
I don't know why that sentence is minus.

**Review by Tabrizian — APPROVED** [review]

**Pull request merged**